lang/hr/index.md

```diff
@@ -0,0 +1,262 @@
+identifikatori MORAJU sastojati samo od ASCII alfanumeričkih znakova i crtica
+[0-9A-Za-z-]. Identifikatori NE SMIJU biti prazni. Numerički identifikatori NE
+SMIJU počinjati nulom. Normalne verzije imaju prednost nad vezanim verzijama
+predizdanja. Oznaka verzije predizdanja označava nestabilnu verziju koja ne
+zadovoljava nužno predviđene zahtjeve kompatibilnosti navedene u vezanoj
+normalnoj verziji. Primjeri: 1.0.0-alpha, 1.0.0-alpha.1, 1.0.0-0.3.7,
+1.0.0-x.7.z.92.
+
+1. Metapodaci builda MOGU se označiti dodavanjem znaka plus i niza identifikatora
+odvojenih točkom, koji se odmah nastavljaju na oznaku verzije zakrpe ili
+predizdanja. Ti se identifikatori MORAJU sastojati samo od ASCII alfanumeričkih
+znakova i crtica [0-9A-Za-z-]. Identifikatori NE SMIJU biti prazni. Metapodaci
+builda se ignoriraju kod procjene prednosti verzije. Dakle, dvije verzije koje
+se razlikuju samo u metapodacima builda imaju jednaku prednost. Primjeri:
+1.0.0-alpha+001, 1.0.0+20130313144700, 1.0.0-beta+exp.sha.5114f85.
+
+1. Pojam prednosti odnosi se na to kako uspoređujemo verzije u poretku.
+Prednost se MORA procjenjivati tako da zasebno promatramo oznaku verzije kao
+veću, manju, zakrpu i identifikatore predizdanja tim redom. (Metapodaci builda
+ne utječu na prednost). Prednost određuje prva razlika u usporedbi svake
+oznake s lijeva na desno na sljedeći način: Oznake veće verzije, manje verzije i
+verzije zakrpe uvijek se uspoređuju numerički. Na primjer: 1.0.0 < 2.0.0 <
+2.1.0 < 2.1.1. Kad su oznake veće, manje i verzije zakrpe jednake, normalna
+verzija ima prednost nad verzijom predizdanja. Primjer: 1.0.0-alpha < 1.0.0.
+Prednost između dvije verzije predizdanja s istim oznakama veće, manje i verzije
+zakrpe MORAJU se odrediti usporedbom svake oznake odvojene točkom s lijeva na
+desno, dok ne dođemo do razlike na sljedeći način: identifikatori koji se
+sastoje samo od broja uspoređuju se numerički, dok se slova i crtice uspoređuju
+leksički u ASCII poretku. Nebrojčane oznake uvijek imaju prednost nad brojčanim.
+Veći niz oznaka predizdanja ima prednost nad manjim nizom, ako su prethodne
+oznake jednake. Primjer: 1.0.0-alpha < 1.0.0-alpha.1 < 1.0.0-alpha.beta < 1.0.0-beta <
+1.0.0-beta.2 < 1.0.0-beta.11 < 1.0.0-rc.1 < 1.0.0.
+
+Zašto koristiti Semantičko verzioniranje?
+----------------------------
+
+Ovo nije nova, niti revolucionarna ideja. Štoviše, vjerojatno već koristite
+nešto vrlo slično. Problem je u tome što "slično" nije dovoljno dobro. Bez
+poštivanja neke formalne specifikacije, oznake verzije su u suštini beskorisne
+za upravljanje ovisnostima. Ako damo ime i jasnu definiciju gore navedenim
+idejama, postaje lakše iskomunicirati svoje namjere korisnicima softvera.
+Jednom kada su te namjere jasne i fleksibilne (ali ne previše fleksibilne)
+možemo definirati specifikacije ovisnosti.
+
+Možemo jednostavnim primjerom pokazati kako Semantičko verzioniranje može
+ostaviti pakao ovisnosti u prošlosti. Zamislimo softversku biblioteku (library)
```

Comment:
The english translation in a brackets seems to me like a good idea. Maybe this should be used throughout the document.

Reply:
switched to italics